mentorship/gsoc/README.md

```diff
@@ -0,0 +1,59 @@
+# Google Summer of Code in Keptn
+
+Google Summer of Code (GSoC) is a global, online program focused on
+bringing new contributors into open source software development.
+GSoC contributors work with an open source organization on a
+summer programming project under the guidance of mentors.
+
+GSoC contributors accepted into the program receive a stipend, paid by Google,
+to work well-defined projects to improve or enhance the Keptn project and its ecosystem.
+In exchange, numerous community members volunteer as "mentors" for GSoC contributors 
+to help integrate them into the open source community and succeed in completing their projects.
+
+## GSoC 2022
+
+- [2022 Project ideas](./2022/project-ideas.md)
+- [Google Summer of Code website](https://summerofcode.withgoogle.com/)
+- [GSoC Timeline](https://summerofcode.withgoogle.com/how-it-works#timeline)
+
+## Participate
+
+We invite GSoC contributors to submit project proposal applications in relation to these ideas.
```

Comment:
Didn't you say that we would also consider proposals that are unrelated to these ideas?  Maybe add "or other issues that interest you" or something?

Reply:
```suggestion
We invite GSoC contributors to submit project proposal applications in relation to these ideas.
Original ideas will be welcome and appreciated too!
```